url6da19b5a7a_krakrak_Adventure_Game_tgz-pJ8-adventure_MainJ8/src/commands/Pick.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package commands;
+
+import items.*;
+import java.util.List;
+
+/**
+ * Command that allows you to take an object that is contained in a room
+ * @author Irakiza Krak
+ */
+public class Pick extends AbstractCommand{
+    
+    //boolean for the if statement below
+    private boolean contains = false;
+    private boolean containsChest = false;
+    private boolean containsChest2 = false;
+    public Pick(){
+        super("GET", "Insert in your backpack object specified in parameters and located in the current room","get");
+    }
+    @Override
+    public void execute(Game game, String params) {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 22. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test.

We recommend that you simplify this method or break it into multiple methods. For example, consider extracting the code block on lines 72-104 into a separate method.